src/index.html

```diff
@@ -2,16 +2,131 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="form-field-last">
+          I accept the term of the agreement
+          <input type="checkbox" required>
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field-last">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="35"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="Cats"
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="Cats"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="timeGoToBed">
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="brandOfCars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field-last">
```

Comment:
why do you need this class?

Reply:
For input type range - when you click on How do you rate our work? the question, it automatically chose range input and u can move it
As far as I remember each input type should have a label

Reply:
you can add the same class as other labels have to have your code unified. Then you can use pseudo-class (`:last-child` or :`not(":last-child)` o apply styles you need)

Reply:
Hi, 
I updated files and decided to use pseudo-class ` :last-child`
Thank you for your comment it really helped me to make this code better and understand how pseudo-class ` :last-child` is working :)